packages/fxa-auth-server/lib/senders/emails/partials/accountDeletionInfoBlock/en.ftl

```diff
@@ -0,0 +1,4 @@
+account-deletion-info-block-communications = If your account is deleted, you’ll still receive emails from { -brand-mozilla-corporation } and { -brand-mozilla-foundation }, unless you <a data-l10n-name="unsubscribeLink">ask to unsubscribe</a>.
```

Comment:
Instead of creating terms here, let's hardcode them. This is what we do elsewhere like Firefox since using them are so uncommon.
Also, in these cases we don't want to use the existing Mozilla brand term either. (e.g. `Mozilla Corporation and Mozilla Foundation` not ` { -brand-mozilla } Corporation`and { -brand-mozilla } Foundation`)

Apart from those reasons, the main reason we probably don't want to use terms is because of issues with fallbacking. If a locale weren't able to translate these emails in time before sending, I'd assume they'd fall back to English. However, terms / strings referenced by a placeholder won't fallback if the referenced term doesn't exist, instead it would just show the placeholder and `{ -brand-mozilla-corporation }` would appear as-is.